music21/tree/fromStream.py

```diff
@@ -182,7 +182,7 @@ def asTree(inputStream, flatten=False, classList=None, useTimespans=False, group
     >>> etFlat.getPositionAfter(0.5)
     SortTuple(atEnd=0, offset=1.0, priority=0, classSortOrder=20, isNotGrace=1, insertIndex=...)
 
-    >>> etFlatNotes = tree.fromStream.asTree(score, flatten=True, classList=[note.Note])
+    >>> etFlatNotes = tree.fromStream.asTree(score, flatten=True, classList=(note.Note,))
```

Comment:
In docs, I tend to pass lists because tuples with trailing commas confuse new programmers.  Here it doesn't matter because beginners shouldn't be working with `tree`, but just FYI.